app/policies/wiki_policy.rb

```diff
@@ -0,0 +1,39 @@
+class WikiPolicy < ApplicationPolicy
+
+  attr_reader :user, :wiki
+
+  def initialize(user, wiki)
+    @user = user
+    @wiki = wiki
+  end
+
+   def index?
+     true
+   end
+
+  def admin_list?
+    user.admin?
+  end
+
+   def create?
+     user.present?
+   end
+
+   def update?
+     user.present? #CURRENT
```

Comment:
Wondering what you were doing here?

Reply:
Again, I was testing different configurations. The checkpoint called for establishing 3 separate roles and with a private feature for the wikis. I kind of jumped the gun and tried to figure out how to implement functionality for these, which will be called for in later checkpoints.